Tests/XCUITests/DatabaseFixtureTest.swift

```diff
@@ -32,73 +32,34 @@ class DatabaseFixtureTest: BaseTestCase {
+        let loaded = NSPredicate(format: "count == 1001")
         expectation(for: loaded, evaluatedWith: app.tables["Bookmarks List"].cells, handler: nil)
-        waitForExpectations(timeout: 60, handler: nil)
+        waitForExpectations(timeout: TIMEOUT_LONG, handler: nil)
 
         let bookmarksList = app.tables["Bookmarks List"].cells.count
-        XCTAssertEqual(bookmarksList, 1013, "There should be an entry in the bookmarks list")
-    }*/
+        XCTAssertEqual(bookmarksList, 1001, "There should be an entry in the bookmarks list")
+    }
 
     func testHistoryDatabaseFixture() throws {
-        throw XCTSkip("MTE-514 Database may not be loaded")
-        /*
-        waitForExistence(app.buttons["urlBar-cancel"], timeout: 15)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        waitForTabsButton()
         navigator.goto(LibraryPanel_History)
-
+        waitForExistence(app.tables["History List"], timeout: TIMEOUT_LONG)
         // History list has one cell that are for recently closed
-        // the actual max number is 100
+        // the actual max number is 101
         let loaded = NSPredicate(format: "count == 101")
         expectation(for: loaded, evaluatedWith: app.tables[AccessibilityIdentifiers.LibraryPanels.HistoryPanel.tableView].cells, handler: nil)
-        waitForExpectations(timeout: 30, handler: nil)
-        */
-    }
-
-    func testPerfHistory4000startUp() {
-        measure(metrics: [
-            XCTMemoryMetric(),
-            XCTClockMetric(), // to measure timeClock Mon
-            XCTCPUMetric(), // to measure cpu cycles
-            XCTStorageMetric(), // to measure storage consuming
-            XCTMemoryMetric()]) {
-            // activity measurement here
-            app.launch()
-        }
-    }
-
-    func testPerfHistory4000openMenu() {
-        measure(metrics: [
-            XCTMemoryMetric(),
-            XCTClockMetric(), // to measure timeClock Mon
-            XCTCPUMetric(), // to measure cpu cycles
-            XCTStorageMetric(), // to measure storage consuming
-            XCTMemoryMetric()]) {
-            // activity measurement here
-            navigator.goto(LibraryPanel_History)
-        }
-    }
-
-    func testPerfBookmarks1000startUp() {
-        measure(metrics: [
-            XCTMemoryMetric(),
-            XCTClockMetric(), // to measure timeClock Mon
-            XCTCPUMetric(), // to measure cpu cycles
-            XCTStorageMetric(), // to measure storage consuming
-            XCTMemoryMetric()]) {
-            // activity measurement here
-            app.launch()
-        }
+        waitForExpectations(timeout: TIMEOUT, handler: nil)
+        let historyList = app.tables["History List"].cells.count
+        XCTAssertEqual(historyList, 101, "There should be 101 entries in the history list")
     }
 
     func testPerfBookmarks1000openMenu() {
+        waitForTabsButton()
```

Comment:
Remove this test from here, it has been moved to the TabsPerformance test suite

Reply:
done